be/src/runtime/exec_env.h

```diff
@@ -200,6 +201,7 @@ class ExecEnv {
 
     // TODO(cmy): find a better way to unify these 2 pools.
     PriorityThreadPool* _scan_thread_pool = nullptr;
+    PriorityThreadPool* _s3_scan_thread_pool = nullptr;
```

Comment:
_s3_scan_thread_pool need to be _remote_scan_thread_pool

Reply:
done